docs/stack/insight/tutorials/insight-stream-consumer.md

```diff
@@ -0,0 +1,199 @@
+---
+title: "Manage streams and consumer groups in RedisInsight"
+linkTitle: Streams
+description: Learn how to manage streams and consumer groups in RedisInsight
+weight: 1
+---
+
+A _stream_ is an append-only log file. 
+When you add data to it, you cannot change it. 
+That may seem like a disadvantage; however, a stream serves as a log or single source of truth. 
+It is used as a buffer between processes that work at different speeds and do not need to know about each other. 
+For more conceptual information about streams, see [Redis Streams](/docs/manual/data-types/streams). 
+
+In this topic, you will learn how to add and work with streams as well as consumer groups in RedisInsight.
+
+Here's a stream that models temperature and humidity sensors. 
+In this stream, sensors are considered _producers_, which broadcast data. 
+The point of a stream is that it's not going to end, so you cannot capture whole datasets and do some processing on them. 
+In order to process the data, a stream is separated into different roles. 
+A _consumer_ reads from the stream and does some work on it. 
+For example, if the temperature is above a certain threshold, it puts a message out to turn on the air conditioner in that unit or notify the maintenance.
+
+<img src="../images/stream.png">
+
+
+It is possible to have multiple consumers doing different jobs, one measuring humidity, and another taking temperature measurements over periods of time. 
+Redis runs everything in memory that is finite. 
+To avoid runaway data, streams can be trimmed when you add something to them. 
+You can specify to remove everything but a number of last entries or remove stream that has an ID lower than specified, for example a timestamp. 
+You can also expire streams using keys by writing to a different key each day and expiring those keys after a period of time, say a week. 
+An ID can be any number.
+
+## Adding new entries 
+
+Use `XADD` with `*` for ID and that automatically generates timestamps plus fields and values. There are a couple of ways of retrieving things. You could give it a time range or you could ask for everything that's happened since a timestamp or ID that you specify. On one command you can ask for anything from 10:30 until 11:15 am on each day over a specified period of time.
```

Comment:
```suggestion
Use `XADD` with `*` for the ID to have Redis automatically generate a new ID for you consisting of a millisecond precision timestamp, a dash and a sequence number.  For example `1656416957625-0`.  Then supply the field names and values to store in the new stream entry.

There are a couple of ways of retrieving things. You can retrieve entries by time range or you could ask for everything that's happened since a timestamp or ID that you specify. Using a single command you can ask for anything from 10:30 until 11:15 am on a given day.
```